### DISCUSSION TITLE
Beta10开机无法自启，软件内可以看到交易记录但不弹窗

### DISCUSSION THREAD

**Discussion by Zori9527** [AutoAccountingOrg/AutoAccounting#305] [求助提问]
我看更新记录里写了Beta10Root模式开机可以自启，但是更新之后重启依旧需要手动去软件启动一次，而且发生交易行为后，去软件内可以看到被识别的交易记录，但是付款后不会自动弹出记账提示，已经给了悬浮窗权限，后台权限也给了

**Comment by AnkioTomas** [comment]
均为已知bug
